src/ext-test/java/org/opentripplanner/ext/emissions/Co2EmissionsDataReaderTest.java

```diff
@@ -0,0 +1,43 @@
+package org.opentripplanner.ext.emissions;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+public class Co2EmissionsDataReaderTest {
+
+  private static final String CO2_GTFS_ZIP_PATH = "src/test/resources/gtfs/emissions-test-gtfs.zip";
+  private static final String CO2_GTFS_PATH = "src/test/resources/gtfs/emissions-test-gtfs";
```

Comment:
Since you opened this PR, we switched to a new way of organising test resources. You must put the test resources in the same package structure as the Java classes and use the `ResourceLoader` class.

Example: https://github.com/opentripplanner/OpenTripPlanner/blob/32bf3d044c9c43c52e794b036f453bd40f4b51d1/src/ext-test/java/org/opentripplanner/ext/flex/FlexTest.java#L25-L34